src/EnergyPlus/StandardRatings.cc

```diff
@@ -2576,14 +2580,21 @@ namespace StandardRatings {
                         << RoundSigDigits(TotElectricPowerRated(Num + 3), 1) << RoundSigDigits(NetCoolingCapRated(Num + 4), 1)
                         << RoundSigDigits(TotElectricPowerRated(Num + 4), 1);
                     PreDefTableEntry(pdchDXCoolCoilType, CompNameNew, CompType);
-                    PreDefTableEntry(pdchDXCoolCoilNetCapSIA, CompNameNew, RoundSigDigits(NetCoolingCapRated(Num + 1), 1));
-                    PreDefTableEntry(pdchDXCoolCoilElecPowerA, CompNameNew, RoundSigDigits(TotElectricPowerRated(Num + 1), 1));
-                    PreDefTableEntry(pdchDXCoolCoilNetCapSIB, CompNameNew, RoundSigDigits(NetCoolingCapRated(Num + 2), 1));
-                    PreDefTableEntry(pdchDXCoolCoilElecPowerB, CompNameNew, RoundSigDigits(TotElectricPowerRated(Num + 2), 1));
-                    PreDefTableEntry(pdchDXCoolCoilNetCapSIC, CompNameNew, RoundSigDigits(NetCoolingCapRated(Num + 3), 1));
-                    PreDefTableEntry(pdchDXCoolCoilElecPowerC, CompNameNew, RoundSigDigits(TotElectricPowerRated(Num + 3), 1));
-                    PreDefTableEntry(pdchDXCoolCoilNetCapSID, CompNameNew, RoundSigDigits(NetCoolingCapRated(Num + 4), 1));
-                    PreDefTableEntry(pdchDXCoolCoilElecPowerD, CompNameNew, RoundSigDigits(TotElectricPowerRated(Num + 4), 1));
+                    // Note JM 2018-10-25: If you call RoundSigDigits(NetCoolingCapRated(Num + 1), 1),
+                    // Then it's not the OutputReportPredefined::PreDefTableEntry prototype with Real64 that is called.
+                    // As a result, the entry isn't marked as being Real (origEntryIsReal) and unit conversion does not occur
+                    // Bad: PreDefTableEntry(pdchDXCoolCoilNetCapSIA, CompNameNew, RoundSigDigits(NetCoolingCapRated(Num + 1), 1));
```

Comment:
Is there a way to check if the string converts to a Real so that this doesn't happen again?

    void PreDefTableEntry(int const columnIndex, std::string const &objName, std::string const &tableEntryChar);

Reply:
@rraustad sorry I feel like it's the responsibility of the dev to ensure he selects the right overloaded prototype, and it would create extra operations to check if it's a real.

Perhaps a compromise might be to use macro code to only do this when E+ is built in `Debug` mode and throw if that's the case?